### ISSUE TITLE
Global filter on ambiences

### ISSUE THREAD

**Issue by bflorat** [jajuk-team/jajuk#411]
**Reported by bflorat on 28 Feb 2007 22:38 UTC**
I explain (for those who don't know what is an ambience, please read       : jajuk.info/index.php/Using_digital_djs :
- In 1.2, ambience selection is done from the DJ drop down button.
- In 1.3, as Jerome suggested, I moved this choice to a new combo in the command panel (see the screenshot)
- For the moment, this combo affects only the global functions at its right (shuffle, novelties and bestof) : if you select "Hard Rock" ambience and then click on Global shuffle button, only Hardrock styles (Trash, Punk, Grunge...)  are launched.
- I see here an opportunity to extend this function to all the app: all views could be affected  by this choice. For instance, in logical tree / table views would be filtered by ambience. You would be get a multi-ambience jukebox: a classical jukebox, a rock jukebox... and one (or all) ambience at the same time !
- However, I wonder if it is not confusing and too complex (some users already report support requests on simpler behaviours like planned tracks).
- and I wonder how far to go if you go this way: filtering physical views as well ? filtering playlists (keeping playlists containing at least one track of this ambience?) ?
- In conclusion there are at least 3 options:
  - 1: Ambience selector affects only the global function. If we need an ambience filter in a view (like in catalog view as asked by a user), we add a view-internal combo each time needed.
  - 2: Ambience selection affects many views (all logical views, catalog) but not physical views
  - 3: Ambience selection affects any views, even physical (but not stats)
    Any suggestion or alternatives ?
